docs/release-notes/release-notes-0.13.995.md

```diff
@@ -4,9 +4,8 @@
 
 ### Lightning Terminal
 
-* [Ensured reproducible
-  releases](https://github.com/lightninglabs/lightning-terminal/pull/852) by
-  setting a fixed timestamps for the files in the release script.
+- [Ensured reproducible releases](https://github.com/lightninglabs/lightning-terminal/pull/852)
```

Comment:
Was that your IDE changing this? Because we're trying to fold things at 80 characters.

Reply:
Yeah, IDE thing.